noxfile.py

```diff
@@ -76,13 +77,14 @@ def test(session):
         run_with_protected_pip(
             session,
             "wheel",
-            "-w", LOCATIONS["common-wheels"],
-            "-r", REQUIREMENTS["common-wheels"],
+            "-w",
+            LOCATIONS["common-wheels"],
+            "-r",
+            REQUIREMENTS["common-wheels"],
         )
     else:
-        msg = (
-            "Re-using existing common-wheels at {}."
-            .format(LOCATIONS["common-wheels"])
+        msg = "Re-using existing common-wheels at {}.".format(
```

Comment:
why not change to f-string?

Reply:
f-string is only available in Python 3.6+.